src/guides/v2.3/migration/migration-tool-internal-spec.md

```diff
@@ -165,6 +165,42 @@ Change prefix option in case Magento was installed with prefix in database table
 
 Configuration data is accessible via \Migration\Config class.
 
+### Steps available operations {#steps}
+
+|Document|Field|
+|---|---|
+|step|Second level node inside the Steps node. Description of the relevant step.|
```

Comment:
We should make it clear that it is the title attribute that takes the description.

Reply:
@dobooth  I have updated the description for this. Please review it and let me know If any changes requires